source/GameAction.cpp

```diff
@@ -190,7 +193,15 @@ void GameAction::LoadSingle(const DataNode &child)
 			swap(minDays, maxDays);
 		events[GameData::Events().Get(child.Token(1))] = make_pair(minDays, maxDays);
 	}
-	else if(key == "fail" && child.Size() >= 2)
+	else if(key == "music" && hasValue && !child.Token(1).empty())
```

Comment:
```suggestion
	else if(key == "music" && hasValue)
```
you dont need this, if somebody stores nothing, they should expect nothing